README.md

```diff
@@ -19,19 +19,18 @@ To integrate NullAway into your non-Android Java project, add the following to y
 ```gradle
 plugins {
   // we assume you are already using the Java plugin
-  id "net.ltgt.errorprone" version "0.6"
+  id "net.ltgt.errorprone" version "<plugin version>"
 }
 
 dependencies {
-  annotationProcessor "com.uber.nullaway:nullaway:0.10.13"
+  errorprone "com.uber.nullaway:nullaway:<NullAway version>"
```

Comment:
Do we want to remove version numbers from here or just adding the version of the plugin to the set of things to update when releasing? Either way, releasing instructions might need updating too!

Reply:
I think we should be consistent and not have the version here.  It hinders copy-paste-ability, but before we were just updating the NullAway version and not versions of other related plugins, and stuff got out of date.  So I propose leaving this and updating the releasing instructions.  WDYT?

Reply:
Fine by me. Another option is to have some script to update the readme that fetches these from our own configuration, or add steps to the release instructions, but that is a bit of work and might not be super reliable anyways (will break if the README ever changes shape significantly). Let's go with these placeholders for not, then.